src/preprocess.ts

```diff
@@ -50,23 +44,22 @@ export function process(
 
   const transpileOutput = transpileTypescript(filePath, src, compilerOptions);
 
+  // FIXME: this should be done in the typescript source, else it's invalidating source maps
```

Comment:
Good catch. Is there already an issue on this?

Reply:
I don't think so! at least did not see one. Tho IDK if comments can break a sourcemap, it needs some investigation, I just put that comment here as I couldn't research while coding, but then forgot it haha!

Reply:
Fair enough. Let's leave as-is for now. I don't think it'll break line-numbers as there's no line-shifts in the replaced code.